### DISCUSSION TITLE
【功能请求】希望增加删除所有用户头像框的功能

### DISCUSSION THREAD

**Discussion by RingingVolcano** [the1812/Bilibili-Evolved#2184] [功能建议 / Ideas]
emmm优先度不太高
总感觉头像框有点杀马特 花里胡哨的（
ABP杀不完 所以希望脚本能实现
![image](https://user-images.githubusercontent.com/72140353/130335937-0c274de8-c57f-4737-92fc-f8b524d75c2d.png)
![image](https://user-images.githubusercontent.com/72140353/130335982-576e4f57-8b56-46ed-85fd-f7458ff6a234.png)

**Comment by HashLiver** [comment]
你不会用ABP，`bilibili.com##.bili-avatar-pendent`即可屏蔽头像饰件。

**Reply by RingingVolcano** [reply]
感谢了
